.ci/pytorch/common_utils.sh

```diff
@@ -135,10 +135,16 @@ function install_filelock() {
 
 function install_triton() {
   local commit
+  commit=$(get_pinned_commit triton)
+  local short_hash
+  short_hash=$(echo "${commit}"|cut -c -10)
+  local index_url
+  index_url=https://donwload.pytorch.org/whl/nightly/cpu
```

Comment:
Typo :) 

```suggestion
  index_url=https://download.pytorch.org/whl/nightly/cpu
```

Reply:
Well, it tested that if package is not found, then it is not found :)
Let me commit the change and check again that it will work if it's found :)